lib/assets.dart

```diff
@@ -1,3 +1,4 @@
 library assets;
 
-export 'src/assets/assets.dart';
+// export 'src/assets/assets.dart';
+export 'src/generated/assets.gen.dart';
```

Comment:
Do we still have the usage of previous AegisAssets in the Athena?

Reply:
Nope, will be deleted